### PR TITLE
1.11 remove not releases hut' blocks

### LINKED ISSUE #789: Quarry etc.
Can we please delete the Quarry and other non implemented buildings out of the mod for now..  The number of times I get told/asked "The quarry doesn't work, please help me" or similar is becoming ridiculous.  I know we have stated "please dont use them for now"etc. but people just dont read/listen to that :)

This includes from the Creative menu etc.

Adding them back in later is a very minor task. 

Thanks,

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/43967250-quarry-etc?utm_campaign=plugin&utm_content=tracker%2F43175463&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F43175463&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by xavierh**
close #789
Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/util/RecipeHandler.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutWareHouse, 1), "XXX", "X#X", "XXX", 'X', PLANK_WOOD, '#', "chest")); // check
         GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(ModBlocks.blockHutDeliveryman, 1), "XXX", "X#X", "XXX", 'X', PLANK_WOOD, '#', Items.LEATHER_BOOTS));
 
-        enableInDevelopmentFeatures(enableInDevelopmentFeatures);
+        //enableInDevelopmentFeatures(enableInDevelopmentFeatures);
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/util/RecipeHandler.java:78]: MINOR This block of commented-out lines of code should be removed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 41 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 40 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [RecipeHandler.java#L82](https://github.com/Minecolonies/minecolonies/blob/8617d7e1311004938138033f6d71d0823e9f6122/src/main/java/com/minecolonies/coremod/util/RecipeHandler.java#L82): Remove this unused private "enableInDevelopmentFeatures" method. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [blockout](https://github.com/Minecolonies/minecolonies/blob/8617d7e1311004938138033f6d71d0823e9f6122/src/main/java/com/minecolonies/blockout): Add a 'package-info.java' file to document the 'blockout' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [controls](https://github.com/Minecolonies/minecolonies/blob/8617d7e1311004938138033f6d71d0823e9f6122/src/main/java/com/minecolonies/blockout/controls): Add a 'package-info.java' file to document the 'controls' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [views](https://github.com/Minecolonies/minecolonies/blob/8617d7e1311004938138033f6d71d0823e9f6122/src/main/java/com/minecolonies/blockout/views): Add a 'package-info.java' file to document the 'views' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/8617d7e1311004938138033f6d71d0823e9f6122/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/8617d7e1311004938138033f6d71d0823e9f6122/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/8617d7e1311004938138033f6d71d0823e9f6122/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/8617d7e1311004938138033f6d71d0823e9f6122/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/8617d7e1311004938138033f6d71d0823e9f6122/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [jobs](https://github.com/Minecolonies/minecolonies/blob/8617d7e1311004938138033f6d71d0823e9f6122/src/main/java/com/minecolonies/coremod/colony/jobs): Add a 'package-info.java' file to document the 'jobs' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by marvin-bitterlich** [comment]
:shipit:

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/972/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/972/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Pull request merged**